lib/vsc/install/shared_setup.py

```diff
@@ -331,7 +332,15 @@ def __init__(self, *args, **kwargs):
                 kwargs['output'] = xmlrunner_output
                 xmlrunner.XMLTestRunner.__init__(self, *args, **kwargs)
 
-        main_orig = unittest.main
+        # old setuptools
+        main_name = 'main'
```

Comment:
i don't understand the remark. it's sufficient to monkey patch setuptools command test, not unittest per se